packages/block-library/src/image/image.js

```diff
@@ -745,6 +762,14 @@ export default function Image( {
 							resetLightbox={ resetLightbox }
 						/>
 					) }
+					{ lightboxChecked && showLightboxSetting && (
+						<ToolbarButton
+							icon={ formatIndent }
+							label={ __( 'Apply Custom Styles to lightbox' ) }
+							onClick={ toggleStylingLightbox }
+							isActive={ lightbox?.applyStyles ?? false }
```

Comment:
In order to maintain existing functionality on existing block-enabled content, the default value should be `true` if `applyStyles` is not already available within the lightbox block settings.
_Correct me if I'm wrong here in case the default value defined in `block.json` is used in this situation._

Reply:
Oh yes, I initially wrote the logic in reverse and then refactored it. I'll update it (though it won't make much of a difference since it defaults to true in block.json).